sass/components/_comments.scss

```diff
@@ -0,0 +1,97 @@
+.container{
+    position: relative;
+}
+
+.img-face{
+    position: absolute;
+    top:0;
+    transform: translateY(-40%);
+    @media (min-width:1120px) {
+        left: -100px;
+    }
+}
+
+.face{
+    margin: 0 auto;
+    margin-bottom: 72px;
+    
+    @media (min-width:768px) {
+        margin-top: 279px;
+    }
+}
+
+.comment{
+    font-family: 'DM Sans';
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 255%;
+    text-align: center;
+    color: #000000;
+    background-color: #FAFAFA;
+    padding: 43px 50px 26px 50px;
+    text-align: center;
+
+    @media screen and (min-width:768px) {
+        background-color:transparent;
+        padding: 0 110px 0 114px;
+        font-size: 14px;
+        line-height: 216%;
+        letter-spacing: 0.04em;
+    }
+    @media (min-width:1120px) {
+        font-size: 16px;
+        line-height: 186%;
+    }
+}
+.comment:before{
+    display:inline-block;
+    content:"";
+    background-image: url(../../image/litter.svg);
+    width: 54px;
+    height: 40px;
+    transform: translateX(20px);
+    opacity: 0.5;
+    
+}
+
+.menu-elipse{
+    display: flex;
+    justify-content: center;
+    margin-top: 30px;
+    margin-bottom: 18px;
+    .item{
+        background-color: #C7133E;
+        width: 4px;
+        height: 4px;
+        border-radius: 50%;
+
+        &:not(:last-child) {
+            margin-right: 10px;
+        }
+    } 
+}
+
+.comment-city{
+    font-family: 'DM Sans';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 186%;
+    text-align: center;
+    color: #000000;
+}
+.menu-elipse.size{
+
+    .item:first-child{
+        background-image: url(../../image/home.png);
+        border-radius: 0;
+        width: 12.4px;
+        height: 11.84px;
```

Comment:
it cannot be less than 1px :) 12 or 13 px( but it is better to use rem)

Reply:
maket krivoy. edit on 12 * 12